fifeforspark/processors.py

```diff
@@ -10,119 +10,131 @@
 findspark.init()
 sc = SparkSession.builder.getOrCreate()
```

Comment:
Same comment. if we refer to every spark session as Spark, I think that would work. I see the merit of putting this in main for sure, but not in this one.